pytorch_lightning/strategies/deepspeed.py

```diff
@@ -463,12 +463,12 @@ def zero_stage_3(self) -> bool:
 
     def _initialize_deepspeed_train(self, model):
         if "optimizer" in self.config:
-            optimizer, lr_scheduler = None, _get_default_scheduler_config()
-        else:
             rank_zero_info(
-                "You have not specified an optimizer or scheduler within the DeepSpeed config."
-                " Using `configure_optimizers` to define optimizer and scheduler."
+                "You have specified an optimizer and/or scheduler within the DeepSpeed config."
```

Comment:
Could it be multiple optimizers?

Reply:
AFAIK, deepspeed only supports a single optimizer/scheduler currently.

Reply:
nope, deepspeed currently only supports one :)